systemtests/src/test/java/io/enmasse/systemtest/TestBase.java

```diff
@@ -439,6 +445,126 @@ protected int getSubscriberCount(AmqpClient queueClient, Destination replyQueue,
         return addresses;
     }
 
+    /**
+     * attach N receivers into one address with default username/password
+     */
+    protected List<AbstractClient> attachReceivers(AddressSpace addressSpace, Destination destination, int receiverCount) throws Exception {
+        return attachReceivers(addressSpace, destination, receiverCount, username, password);
+    }
+
+    /**
+     * attach N receivers into one address with own username/password
+     */
+    protected List<AbstractClient> attachReceivers(AddressSpace addressSpace, Destination destination, int receiverCount, String username, String password) throws Exception {
+        ArgumentMap arguments = new ArgumentMap();
+        arguments.put(Argument.BROKER, getRouteEndpoint(addressSpace).toString());
+        arguments.put(Argument.TIMEOUT, "180");
```

Comment:
180 is too much, default executor timeout for async client is 120, this timeout is like fetch.

Reply:
ah, thanks, it was for my debugging purpose -> fixed